subsys/esb/esb_dppi.c

```diff
@@ -219,7 +252,8 @@ void esb_ppi_disable_all(void)
 				  BIT(egu_timer_start) |
 				  BIT(radio_address_timer_stop) |
 				  BIT(timer_compare0_radio_disable) |
-				  BIT(timer_compare1_radio_txen));
+				  IF_ENABLED(CONFIG_ESB_NEVER_DISABLE_TX, BIT(timer_compare1_radio_txen) |)
```

Comment:
I suggest change an order to put this `IF_ENABLED` at the end and also second parameter of this macro must be in brackets

```
BIT(radio_end_timer_start) |
 IF_ENABLED(CONFIG_ESB_NEVER_DISABLE_TX, (BIT(timer_compare1_radio_txen))));
```

Reply:
Done